subsys/infinispan/src/main/resources/infinispan.xml

```diff
@@ -32,10 +32,15 @@
       </indexing>
     </local-cache>
 
+    <local-cache name="metadata-cache" deadlock-detection-spin="10000">
```

Comment:
Can we make this maven-version-metadata-cache? That way we have room for maven-archetype-metadata-cache, npm-package-metadata-cache, etc.

Reply:
Will change.